public/components/datasources/components/data_connections_header.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiLink,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiSpacer,
+  EuiTab,
+  EuiTabs,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { OPENSEARCH_DOCUMENTATION_URL } from '../../../../common/constants/data_connections';
+
+export const DataConnectionsHeader = () => {
+  const tabs = [
+    {
+      id: 'manage',
+      name: 'Manage data source',
+      disabled: false,
+    },
+    {
+      id: 'new',
+      name: 'New data source',
+      disabled: false,
+    },
+  ];
+
+  const [selectedTabId, setSelectedTabId] = useState(
+    window.location.hash.substring(2) ? window.location.hash.substring(2) : 'manage'
```

Comment:
what is this?

Reply:
This is to toggle between which of the tabs are open, based on the url hash. Not sure if this is a good practice or if the state should just be managed internally/lost on page refresh - wdyt @joshuali925 ?

Reply:
i've not tested this, but proper way is probably through `useLocation()`

Reply:
Done in: [680763b](https://github.com/opensearch-project/dashboards-observability/pull/1049/commits/680763b566a24a70d93f79214331182f495ed291)